hackathon_site/event/api_urls.py

```diff
@@ -20,4 +20,9 @@
         name="team-orders",
     ),
     path("teams/<int:pk>/", api_views.TeamDetailView.as_view(), name="team-detail"),
+    path(
+        "teams/team/order/<int:pk>/",
+        api_views.TeamOrderDetailView.as_view(),
+        name="order-detail",
```

Comment:
```suggestion
        name="team-order-detail",
```

I'm not actually sure if having a unique name is required since the other `order-detail` view is in the hardware app. I don't think so, but doesn't hurt to add clarity.